src/pieces.rs

```diff
@@ -125,8 +123,8 @@ pub fn initial_hand() -> Vec<PieceCombination> {
         PawnBronze,
         PawnSilver,
         PawnGold,
-    ].to_vec();
-    return vec;
+    ];
+    vec
```

Comment:
You could just directly return the vec instead of putting it in a temporary then immediately returning it